### PR TITLE
Enable the 'Share' button when there is a single work

### PR DESCRIPTION
**Pull request by jcoyne**
This is a backport of #837

### CODE REVIEW

File: app/views/hyrax/homepage/index.html.erb [hunk 1]
```diff
@@ -4,10 +4,18 @@
...
-          <i class="glyphicon glyphicon-upload"></i> <%= t('hyrax.share_button') %>
+        <% if create_work_presenter.many? %>
+          <%= link_to '#',
+            class: "btn btn-primary btn-lg",
+            data: { behavior: 'select-work', target: '#worktypes-to-create', 'create-type' => 'single' } do %>
+            <i class="glyphicon glyphicon-upload"></i> <%= t('hyrax.share_button') %>
+          <% end %>
+        <% else # simple link to the first work type %>
+          <%= link_to(
+                t('hyrax.share_button'),
```
Comment (rotated8) [inline app/views/hyrax/homepage/index.html.erb]: Does this version of the link need the glyphicon part?

Comment (jcoyne) [inline app/views/hyrax/homepage/index.html.erb]: @rotated8 your bug report. What do you think?

Comment (rotated8) [inline app/views/hyrax/homepage/index.html.erb]: I figure it should, for consistency, but I don't have strong feelings about it.

Comment (mjgiarlo) [inline app/views/hyrax/homepage/index.html.erb]: Consistency++

**Pull request merged**